charms/candid/tests/data/test-bundle.yaml

```diff
@@ -0,0 +1,67 @@
+applications:
```

Comment:
Maybe rename this bundle for future if we have other bundles, possibly prefux with a number and purpose?

Reply:
done